README.md

```diff
@@ -10,11 +10,368 @@
+  password should be provided in a separate file via the "dbPasswordFile" config
+  parameter.
+
+* **dbPasswordFile** - a filepath for a file containing the DB user password.
+  _NOTE_: File must have mode `0600`.
+
+* **fetchConfig** - global configuration related to periodic STH fetching.
+
+  * **interval** - a duration string describing the time period between fetching STHs
+
+  * **timeout** - a duration string describing the timeout for fetching an STH.
+
+* **submitConfig** - global configuration related to periodic cert issuance and
+  submission. May be omitted.
+
+  * **interval** - a duration string describing the time period between attempts
+    to issue and submit certs/precerts.
+
+  * **timeout** - a duration string describing the timeout for submitting
+    a cert/precert.
+
+  * **certIssuerKeyPath** - a filepath for a file containing a PEM encoded
+  RSA/ECDSA private key corresponding to the public key in the
+  **certIssuerPath** PEM encoded intermediate certificate.
+
+  * **certIssuerPath** - a filepath for a file containing a PEM encoded x509
+  certificate to use as the issuer for certificates generated for submitting
+  to logs.
+
+* **inclusionConfig** global configuration related to checking that certificates
+  issued periodically by `ct-woodpecker` were included in the monitored logs.
+
+  * **interval** - a duration string describing the time period between attempts
+    to check unseen certificates for inclusion.
+
+  * **maxGetEntries** - the maximum number of cert entries to ask a log for.
+  This should be set quite high as `ct-woodpecker` will reduce the number of
+  certs it asks for based on what the log will actually provide.
+
+  * **startIndex** - an optional integer specifying the treesize to start
+  checking for inclusion from. This is useful if you start `ct-woodpecker`
+  monitoring against a log that already has a large tree to let `ct-woodpecker`
+  skip ahead to the `startIndex`.
+
+* **logs** - an array of one or more CT logs to be configured. Each log is
+  composed of a config object with the following fields:
+
+  * **uri** - the log's URI.
+
+  * **key** - the log's public key (PEM encoded as a single line without the PEM
+  header/footer).
+
+  * **windowStart** - (optional) for a sharded log the `windowStart` specifies
+  the begin date for the shard's accepted validity window. `ct-woodpecker` will
+  ensure the certificates it generates for this log have a validity period
+  within the `windowStart` and `windowEnd`
+
+  * **windowEnd** - (optional) for a sharded log the `windowEnd` specifies
+  the end date for the shard's accepted validity window. `ct-woodpecker` will
+  ensure the certificates it generates for this log have a validity period
+  within the `windowStart` and `windowEnd`
+
+  * **maximum_merge_delay** - the MMD for the log.
+
+  * **submitPreCert** - if true then poisoned precertificates for this log will
```

Comment:
```suggestion
  * **submitPreCert** - if true then precertificates for this log will
```

(`poisoned` is part of the definition of precertificates so we can leave it out)